packages/table/src/common/loadableContent.tsx

```diff
@@ -29,7 +29,7 @@ export class LoadableContent extends React.Component<ILoadableContentProps, {}>
 
     public constructor(props: ILoadableContentProps) {
         super(props);
-        this.calculateStyle(props.loading);
+        this.calculateStyle(props.variableLength);
```

Comment:
preferable to have this function return the styles instead of setting them internally (#sideeffect).

`this.style = this.calculateStyle()`